eva_cttv_pipeline/evidence_string_generation/clinvar.py

```diff
@@ -20,28 +19,40 @@ class ClinvarRecord(UserDict):
     }
 
     def __init__(self, cellbase_dict):
+        """Initialise a ClinVar record object from JSON data. See /clinvar-variant-types/README.md for the in-depth
+        explanation of ClinVar data model. See also issue #127 for the most recent discussions on changing support of
```

Comment:
maybe say that #127 is a github issue? these comments will stay for years.

Reply:
Good point, thanks! Fixed in https://github.com/EBIvariation/eva-opentargets/pull/135/commits/30cec20cf24a27534f25108d81a806a0435e3192